roles/marathon/defaults/main.yml

```diff
@@ -1,8 +1,30 @@
 ---
 # defaults file for marathon
-marathon_port: 8080
-consul_dir: /etc/consul.d
+marathon_enabled: true
+marathon_version: 'v0.8.2'
```

Comment:
Would need to rebuild packer images if we dont want it to go the internet though